tests/test_path.py

```diff
@@ -37,3 +41,45 @@ def test_path(system_Haldane_PythTB):
                                                   [3, 3, 3],
                                                   [4, 4, 4],
                                                   [5, 5, 5],]) / 10), "path.K_list is wrong"
+
+
+    quantities = quantities_tab
```

Comment:
Separate this part to a new function `test_path_tabulate` (because `test_path` focuses on the construction of the path itself).

Reply:
Done. Made construction as a ixture.